app/views/users/edit.html.erb

```diff
@@ -79,9 +79,18 @@ minlength: User.password_length.min %>
      <%= form.label :receive_email_notifications, t("communication_preferences.email_me") %>
    </div>
    <div>
-     <%= form.check_box :receive_sms_notifications, class: "toggle-sms-notifications" %>
+     <%= form.check_box :receive_sms_notifications, id: "toggle-sms-notifications", class: "toggle-sms-notifications" %>
      <%= form.label :receive_sms_notifications, t("communication_preferences.text_me") %>
    <div>
+   <div class="pl-4 pb-4">
+      <%= form.collection_check_boxes("sms_notification_event_ids", SmsNotificationEvent.where(user_type: @user.type),
+:id, :name) do |event| %>
+        <div class="form-check">
+          <%= event.check_box(class: "form-check-input", id: "toggle-sms-notification-event") %>
```

Comment:
This is looking ahead but as the number of notification events grow adding a drop-down/collapse menu could be good at some point. Probably in the same vein as password confirms dropdown. Checkbox is ticked → menus drops down with event checkboxes. checkbox is unticked → menu collapses

Reply:
good idea